package.json

```diff
@@ -21,5 +22,38 @@
   "bugs": {
     "url": "https://github.com/WTW-IM/react-html-element/issues"
   },
-  "homepage": "https://github.com/WTW-IM/react-html-element#readme"
+  "homepage": "https://github.com/WTW-IM/react-html-element#readme",
+  "lint-staged": {
+    "src/**/*": [
+      "prettier --single-quote --write",
+      "eslint --parser-options \"{project: './tsconfig.json'}\" --fix"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "prettier": {
+    "singleQuote": true
+  },
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^2.24.0",
+    "@typescript-eslint/parser": "^2.24.0",
+    "eslint": "^6.5.1",
+    "eslint-config-airbnb-typescript": "^7.2.0",
+    "eslint-config-prettier": "^6.10.0",
+    "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.19.0",
+    "eslint-plugin-react-hooks": "^2.5.1",
+    "husky": "^4.2.3",
+    "lint-staged": "^10.0.8",
+    "prettier": "1.19.1",
+    "typescript": "^3.8.3"
+  },
+  "dependencies": {
+    "react": "^16.13.0",
+    "react-dom": "^16.13.0"
```

Comment:
Let's add these as dev dependencies and peer dependencies (probably just backwards compatible with all of v16) instead of direct dependencies. Don't wanna have conflicting versions of React packages installed downstream.

Reply:
Done!